setup.py

```diff
@@ -84,7 +84,7 @@ def run(self):
 # Extra dependencies for remote integrations
 
 gs = ["google-cloud-storage==1.19.0"]
-gdrive = ["pydrive2>=1.4.6"]
+gdrive = ["pydrive2>=1.4.6", "google-api-python-client >= 1.7.12"]
```

Comment:
we don't probably need this? pydrive installs the client anyway, HttpError should be available

Reply:
see my comment above, disregard this one